patches/minecraft/net/minecraft/world/level/block/entity/BrewingStandBlockEntity.java.patch

```diff
@@ -1,5 +1,24 @@
 --- a/net/minecraft/world/level/block/entity/BrewingStandBlockEntity.java
 +++ b/net/minecraft/world/level/block/entity/BrewingStandBlockEntity.java
+@@ -25,6 +_,7 @@
+ import net.minecraft.world.level.block.state.BlockState;
+ 
+ public class BrewingStandBlockEntity extends BaseContainerBlockEntity implements WorldlyContainer {
++   private static final net.minecraftforge.common.BrewingFuelValues BREWING_FUELS = net.minecraftforge.common.ForgeHooks.onBrewingFuelRegisterEvent();
```

Comment:
This again has the static initializer issue. But also you're firing the event twice, once here and once in the menu. Better is to find a way to fire it once and use it in both places.

Reply:
I resolved this by making `BrewingFuelValues` work in a static way, like how `BrewingRecipeRegistry` does. This should make things simpler for both players and maintenance.